docs/source/usage.rst

```diff
@@ -82,17 +83,27 @@ PDF
 
   Generates a PDF via latex. Supports the same templates as ``--to latex``.
 
+.. _convert_webpdf:
+
+WebPDF
+~~~~~~
+* ``--to webpdf``
+
+  Generates a PDF via puppeteer (chromium web browser). Supports the same
```

Comment:
would again remove reference to pupeteer. Chromium should probably be capitalized.